packages/design-tokens/src/getters.ts

```diff
@@ -20,16 +13,24 @@ import {
   TypographySize,
   ZIndexGroup,
   StyledComponentsInterpolation,
+  Theme,
 } from './types'
 
-export function getBreakpoint(size: BreakpointSize): Breakpoint | undefined {
+function getTheme(theme?: Theme): Theme {
+  return theme?.coloursTokens ? theme : defaultTheme
+}
+
+export function getBreakpoint(
+  size: BreakpointSize,
+  theme?: Theme
+): Breakpoint | undefined {
   const breakpoint = get(
```

Comment:
I wonder how this could be written so that we are not going `getTheme(theme).breakpointsTokens` twice.. and also  `breakpoint[${size}].x.value` twice.

Along the lines of:

```
const { breakpointsTokens } = getTheme(theme)
```